quickstart/docker/image/ziti-cli-functions.sh

```diff
@@ -127,31 +127,42 @@ function checkHostsFile {
   fi
 }
 
-function getLatestZitiVersion {
+function getZitiVersion {
```

Comment:
why this name change? I'm not averse to it but i'd just rather keep these sorts of changes to a minimum. i wonder if anyone else uses this. i think i used it on some blog / article at one point... better to jsut keep it. i'd prefer to provide a "fetch this exact version of ziti" function ( i see below you are allowing for the version override )